scholia/app/templates/venue.html

```diff
@@ -39,7 +39,7 @@
 
  prolificAuthorsSparql = `
 # Prolific authors for a specific journal
-SELECT ?count ?author ?authorLabel ?orcid
+SELECT ?count ?author ?authorLabel ?ORCID
```

Comment:
I usually use lower case for variables, - even variables that refer to identifiers that are usually written with capital letter. In this case it does not work with scholia.js: https://github.com/fnielsen/scholia/blob/master/scholia/app/static/scholia.js#L27